docs/cue/reference/examples.cue

```diff
@@ -0,0 +1,84 @@
+package metadata
```

Comment:
Nice 👍 

It should be possible to wire these up with the tests in `lib/vrl/tests` to ensure they run and don't break if you feel inspired.

Reply:
Hmmm. As far as I know we don't have the facilities to test full end-to-end Vector configs like this. I was thinking, though, that I might want to add a separate VRL section to the main page, and I do think the VRL tests could be wired up to that.

Reply:
Yeah, you are right. I neglected these were full configs, not just VRL snippets. We could at least run them through `vector validate --no-environment` though to catch simple errors.